benches/function.rs

```diff
@@ -10,14 +10,15 @@ fn main() {
   v8::V8::initialize_platform(platform);
   v8::V8::initialize();
   let isolate = &mut v8::Isolate::new(v8::CreateParams::default());
-  let handle_scope = &mut v8::HandleScope::new(isolate);
+  let handle_scope = std::pin::pin!(v8::HandleScope::new(isolate));
+  let handle_scope = &mut handle_scope.init();
```

Comment:
Can we add a helper macro that would do all of this in a single call?

Reply:
Yeah there is one, I just started migrating things over before I added the macro. I've moved more to use the macro now